### DISCUSSION TITLE
Is Nested remotes possible with dynamic runtime remotes

### DISCUSSION THREAD

**Discussion by DeveshSapkale** [module-federation/module-federation-examples#4321] [Q&A]
Hello,

I am interested in understanding whether module federation runtime can accommodate nested remotes. 

To illustrate, consider the following structure:
- MFE 1 serves as the host.
- MFE 2 functions as a remote for MFE 1 while also acting as a host for MFE 3.
- MFE 3 is a remote that MFE 2 will utilize.

I intend to implement {registerRemotes, loadRemote} from @module-federation/enhanced/runtime for this setup. Is it feasible to achieve this with module federation v2, and does it inherently support such a configuration? Additionally, are there any existing examples that demonstrate this? I came across one example of nested remotes, but it utilized a build-time plugin (nested-remote).

I hope I have clearly conveyed my inquiry. 

Thank you,  
@ScriptedAlchemy

**Comment by ScriptedAlchemy** [comment]
Shouldn't be a problem.

**Reply by DeveshSapkale** [reply]
Okay, I will try to create a POC and will get back if something goes wrong.
